lib/Cargo.toml

```diff
@@ -14,14 +24,14 @@ p256 = ["ssi/p256", "did-tz/p256", "did-method-key/p256"]
 
 [dependencies]
 #didkit_cbindings = { path = "cbindings/" }
-ssi = { path = "../../ssi", default-features = false }
-did-method-key = { path = "../../ssi/did-key" }
-did-tz = { path = "../../ssi/did-tezos", default-features = false }
-did-ethr = { path = "../../ssi/did-ethr" }
-did-pkh = { path = "../../ssi/did-pkh" }
-did-sol = { path = "../../ssi/did-sol" }
-did-web = { path = "../../ssi/did-web" }
-did-onion = { path = "../../ssi/did-onion" }
+ssi = { version = "0.2", path = "../../ssi", default-features = false }
```

Comment:
Did we want to use the `ssi` package on crates.io?

Reply:
not sure if it can be done, but it wouldn't be nice if we first check if it exists on this path and as fallback go to crates?

Reply:
This change makes didkit use the ssi package on crates.io when didkit is stilled from crates.io. When it is used locally, it will still require `ssi` to be cloned alongside the `didkit` repo. I don't think we can make the use of the local path conditional on the path existing (although I thought that was how it works, testing now shows it is not the case).

We could decouple `didkit` from `ssi` even for local git use, by removing the `path = ` parts here. This could make it easier for people developing didkit, to not need `ssi`. But this could make it a little harder to develop didkit and ssi at the same time (having to edit `Cargo.toml` to add back the `path =` parts). I don't this question is blocking these changes, since these changes are incremental and do not break the existing usage. 

Reference: <https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations>

> The git or path dependency will be used locally (in which case the version is checked against the local copy), and when published to a registry like crates.io, it will use the registry version.

Reply:
Opened issue for this: https://github.com/spruceid/didkit/issues/141

Reply:
I'm okay keeping this for now, thanks for making the issue!